geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloFeatureWriterTest.scala

```diff
@@ -434,7 +434,7 @@ class AccumuloFeatureWriterTest extends Specification with TestWithDataStore wit
       // ensure that the second part of the UUID is random
       rowKeys.map(_.substring(19)).toSet must haveLength(5)
 
-      val ids = rows.map(e => serializer.deserialize(e.getValue.get).getID)
+      val ids = rows.map(e => RecordTable.getIdFromRow(sft)(e.getKey.getRow))
```

Comment:
yeah, it's used in various places. There was already a method like it in most of the GeoMesaTable classes, I just standardized it as part of the interface.